### PR TITLE
New package: xzoom-0.3

### PR DESCRIPTION
**Pull request by dan-santana**

### CODE REVIEW

File: srcpkgs/xzoom/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+distfiles="http://webdiis.unizar.es/pub/unix/X11/${pkgname}-${version}.tgz"
+checksum=b26bfd692d3e1d30e8d524d4018fc3f0fe847983a2778a4a2496784bfc2df44b
+patch_args="-Np1"
+
+do_build() {
+	CC=cc xmkmf
+	sed -i "s@-O2@$CFLAGS@" ./Makefile
+	make CC=$CC CDEBUGFLAGS="${CFLAGS}" EXTRA_LDOPTIONS="${LDFLAGS}"
+}
+
```
Comment (maxice8) [inline srcpkgs/xzoom/template]: extra line

File: srcpkgs/xzoom/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'xzoom'
+pkgname=xzoom
+version=0.3
+revision=1
+hostmakedepends="imake"
+makedepends="libXt-devel libXext-devel"
+short_desc="Magnify part of X display, with real-time updates"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="Custom"
+homepage="ftp://sunsite.unc.edu/pub/linux/libs/X/"
```
Comment (maxice8) [inline srcpkgs/xzoom/template]: please avoid ftp:// pages

File: srcpkgs/xzoom/template (hunk 1 above)
Comment (dan-santana) [reply srcpkgs/xzoom/template]: fixed

File: srcpkgs/xzoom/patches/xzoom_0.3-23.patch [hunk 3]
```diff
@@ -0,0 +1,928 @@
...
+-The Ximage data is accessed directly, in a way which may
+-not be portable. Xzoom will not run with display depth other
+-than 8 bits per pixel.
+-.LP 5
+-\(dg
+ Xzoom is given with no warranty. It was tested only under
+ Linux with Xfree86 release 3.1.2 (X11R6).
+ .LP 5
+--- xzoom-0.3.orig/debian/copyright
++++ xzoom-0.3/debian/copyright
```
Comment (maxice8) [inline srcpkgs/xzoom/patches/xzoom_0.3-23.patch]: this is not needed in the patch

Comment (dan-santana) [reply srcpkgs/xzoom/patches/xzoom_0.3-23.patch]: fixed

**Comment by dan-santana** [comment]
Anything missing in this package?

**Pull request closed without merging**